src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -2149,11 +2149,7 @@ public static boolean isPlayerOnline( final String player )
 	{
 		public void run()
 		{
-			if ( KoLConstants.VERSION_NAME.startsWith( "KoLmafia r" ) )
-			{
-				return;
-			}
-
+			/* TODO: Check for new version on jenkins\github after migrated
```

Comment:
I personally view commenting out version-controlled code as an anti-pattern, since if you really need it, you can always grab it from a past commit.

Do you intend to merge this change in the next couple of days? If not, maybe just add the new implementation? If so, it's okay to just leave a TODO.

Reply:
Yeah the intent is for this to get merged on the switch to github, then once git releases are working fine I'll rewrite this to find the latest jar from git or jenkins, but since we don't support auto-updating the "daily" builds at the moment I was going to do it in a separate commit and put the checking begind a global var. I'll rip the commented code out and leave the TODO.